src/php/ext/grpc/php7_wrapper.h

```diff
@@ -56,6 +56,14 @@
 #define PHP_GRPC_WRAP_OBJECT_END(name) \
   } name;
 
+#define PHP_GRPC_FREE_WRAPPED_FUNC_START(klass) \
+  void free_##klass(void *object TSRMLS_DC) { \
+    klass *p = (klass *)object;
```

Comment:
Because the name of function end with class name, like `free_wrapped_grpc_timeval`, etc, so I use `klass`.